python/lsst/ts/genericcamera/genericcameracsc.py

```diff
@@ -491,16 +605,28 @@ async def take_image(
             await self.camera.end_shutter_open()
             await self.evt_endShutterOpen.write()
 
+        # Put start integration timestamp closer to call
+        timestamp = utils.current_tai()
```

Comment:
So, after looking at the code further down, I realized that this is used later to report the time when the integration started. I think it might be a good idea to mention this in the comment so future selves know why we are keeping this particular timestamp. Maybe say something like:

> Put start integration timestamp closer to call.
> This value is used later (in endReadout) so we need to keep it until then.

Reply:
OK. I'll add your extra comment.

Reply:
I think you meant `startReadout` as that's where it's used again. `endReadout` uses the timestamp from the camera driver.

Reply:
We use this in all the events so I think the comment needs to say something along those lines.

Reply:
Done.